test/integration/sequelize/query.test.js

```diff
@@ -342,6 +342,20 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
           await this.UserVisit.sync({ force: true });
         });
 
+        it('emits raw errors if requested', async function() {
+          const sql = 'SELECT 1 FROM NotFoundTable';
+          try {
+            await this.sequelize.query(sql, { rawErrors: false });
+          } catch (err) {
+            expect(err).to.be.instanceOf(DatabaseError);
+          }
+          try {
+            await this.sequelize.query(sql, { rawErrors: true });
```

Comment:
Should we make sure that we actually hopped into the catch block? expect.fail in the next line?

Reply:
fixed